getting_started.md

```diff
@@ -0,0 +1,109 @@
+    1. Define the set of feasible points by defining a value for "feasible_points" for the hyperparameters:
+        ```
+        {
+            "learning_rate": {
+                "feasible_points": 0.999
+                },
+        }
+        ```
+        For a complete example see [tuning_search_space.json](https://github.com/mlcommons/algorithmic-efficiency/blob/main/reference_algorithms/target_setting_algorithms/imagenet_resnet/tuning_search_space.json).
+
+    2. Define a range of values for halton sampling by specifing a "min", "max" and "scaling" 
+        keys for the hyperparameter:
+        ```
+        {
+            "weight_decay": {
+                "min": 5e-3, "max": 1.0, "scaling": "log"
+                }
+        }
+        ```
+        For a complete example see[tuning_search_space.json](https://github.com/mlcommons/algorithmic-efficiency/blob/main/baselines/nadamw/tuning_search_space.json). 
+
+
+## Run your submission
+TODO(kasimbeg): Add script to run all workloads?
+
+You can evaluate your submission with the `submission_runner.py` module on one workload at a time. 
+To run 
+
+### JAX submissions
+To score your submission on a workload, from the algorithmic-efficency directory run: 
+```bash
+python3 submission_runner.py \
+    --framework=jax \
+    --workload=mnist \
+    --experiment_dir=<path_to_experiment_dir>\
+    --experiment_name=<experiment_name> \
+    --submission_path=<path_to_submission_module> \
+    --tuning_search_space=<path_to_tuning_search_space>
+```
+
+### PyTorch submissions
+To score your submission on a workload, from the algorithmic-efficency directory run: 
+```bash
+python3 submission_runner.py \
+    --framework=pytorch \
+    --workload=<workload> \
+    --experiment_dir=<path_to_experiment_dir> \
+    --experiment_name=<experiment_name> \
+    --submission_path=<path_to_submission_module> \
+    --tuning_search_space=<path_to_tuning_search_space>
+```
+
+#### PyTorch DDP
+We recommend using PyTorch's [Distributed Data Parallel (DDP)](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html) when using multiple GPUs on a single node. You can initialize ddp with torchrun. 
+For example, on single host with 8 GPUs simply replace `python3` in the above command by:
+```bash
+torchrun --redirects 1:0,2:0,3:0,4:0,5:0,6:0,7:0 --standalone --nnodes=1 --nproc_per_node=N_GPUS
+```
+So the complete command is:
+```bash
+torchrun --redirects 1:0,2:0,3:0,4:0,5:0,6:0,7:0 \
+    --standalone \
+    --nnodes=1 \
+    --nproc_per_node=N_GPUS \
+    submission_runner.py \
+    --framework=pytorch \
+    --workload=<workload> \
+    --experiment_dir=<path_to_experiment_dir> \
+    --experiment_name=<experiment_name> \
+    --submission_path=<path_to_submission_module> \
+    --tuning_search_space=<path_to_tuning_search_space>
+```
+
+### Running submission through Docker
+Todo(kasimbeg)
+
```

Comment:
We also need to include an example of how to compute their score, and ideally how to plot the performance profiles

Reply:
Will work on this in follow up PR, because current scoring utility methods are not convenient for submitters, i.e. they assume we have the log output, which is not automatically saved yet. For tracking: https://github.com/mlcommons/algorithmic-efficiency/issues/413